app/Domain/Users/Templates/editOwn.blade.php

```diff
@@ -313,14 +313,16 @@ class="button"
                             <input type="hidden" name="{{ session("formTokenName") }}" value="{{ session("formTokenValue") }}" />
                             <div class="row-fluid">
                                 <div class="form-group">
-                                    <label for="notifications" >{{ __('label.receive_notifications') }}</label>
-                                    <span>
-                                        <input type="checkbox" value="" name="notifications" class="input"
-                                               id="notifications"
-                                               @if ($values['notifications'] == "1" )
-                                                   checked='checked'
-                                               @endif/> <br/>
-                                    </span>
+                                    <x-global::forms.checkbox
+                                        name="notifications"
+                                        id="notifications"
+                                        value=""
```

Comment:
Can you remove the empty value attribute? This is not needed.

Reply:
done